_sass/pages/_repl.scss

```diff
@@ -194,6 +194,8 @@
 
 .babel-repl-v2 {
   height: 100%;
+  margin-bottom: $padding-large-vertical * -4; // Compensate for _footer.scss margin-top
```

Comment:
I thought the comment explained this, but `_footer.scss` defines:
```css
margin-top: $padding-large-vertical * 4;
```

This offsets that. The top margin doesn't look right in between the footer and the REPL.